docs/api.md

```diff
@@ -586,8 +578,8 @@ const main = async () => {
   // "transformers" field expects array of functions to operate update the code.
   // We will pass single operator "builtInMethods" provided by the framework
   const transformers = [builtInMethods];
-  const result = await executeScript({ code, transformers });
-  console.log({ result });
+  const [result, error] = await executeScript({ code, transformers });
+  console.log(result, error);
```

Comment:
More curly braces! 😄

Reply:
Changed into the following format: `console.log({ blockOffset }, { err })`. Please let us know if this isn't the format you are looking for.